tools/arriba.wdl

```diff
@@ -137,10 +137,14 @@ task arriba {
         File? annotate_fusions
         File? protein_domains
         File? wgs_svs
-        Array[String] interesting_contigs = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15", "16", "17", "18", "19", "20", "21", "22", "X", "Y", "AC_*", "NC_*"]
+        Array[String] interesting_contigs = flatten(
+            [range(23), ["X", "Y", "AC_*", "NC_*"]]
```

Comment:
https://github.com/openwdl/wdl/blob/wdl-1.1/SPEC.md#range

This is going to include `0`, so this needs a patch before we can merge